### PR TITLE
fix(issues): Fix loading display

### PR DESCRIPTION
**Pull request by lynnagara**
Prevent footer floating in the middle of page when loading. Also removes
an unnecessary feature check that was moved into the shared group
details component.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ const GroupDetails = createReactClass({
         // https://github.com/reactjs/react-router/blob/v2.0.1/modules/index.js#L25
         if (this.props.params.groupId != data.id) {
           const location = this.props.location;
-          return void browserHistory.push(
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx]: we don't need this return? seems like other code will execute that didn't before?

File: src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx [hunk 2]
```diff
@@ -254,7 +254,9 @@ const GroupDetails = createReactClass({
                 />
               )}
               {isLoading ? (
-                <LoadingIndicator />
+                <PageContent>
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx]: 👏

**Review by Chrissy — APPROVED** [review]
lgtm % the return thing.

File: src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx]: Ah, thank you

**Pull request merged**